code/game/turfs/open/_open.dm

```diff
@@ -52,22 +52,22 @@
 			. += mutable_appearance(damaged_dmi, pick(broken_states()))
 
 //direction is direction of travel of A
-/turf/open/zPassIn(atom/movable/A, direction, turf/source)
+/turf/open/zPassIn(direction)
 	if(direction == DOWN)
 		for(var/obj/O in contents)
 			if(O.obj_flags & BLOCK_Z_IN_DOWN)
 				return FALSE
 		return TRUE
 	return FALSE
```

Comment:
do you wanna fix this to have early returns like you do `zPassOut()` a few lines down? probably optional but it'd be nice if both of these rather similar procs had the same code quality

Reply:
I honestly just didn't notice sure